.github/workflows/ett.yml

```
@@ -138,13 +138,13 @@ jobs:
       - uses: actions/download-artifact@v2
         with:
           name: ${{ env.app-name }}
-          path: ./dist/apps/${{ env.app-name }}/
+          path: ./dist/apps/ett/${{ env.app-name }}/
 
       - name: Container image name
         id: image-name
         uses: ASzc/change-string-case-action@v2
         with:
-          string: ghcr.io/${{ github.repository }}-${{ env.app-name }}
+          string: ghcr.io/${{ github.repository }}-ett-app
```

suggestion: let's add a comment for me to solve
```suggestion
          # TODO(xlgni): coordinate name change to `env.app-name` with Fenris infrastructure
          string: ghcr.io/${{ github.repository }}-ett-app
```

---

Good point! :D